modules/apps/collaboration/wiki/wiki-web/src/main/resources/META-INF/resources/wiki/edit_page.jsp

```diff
@@ -217,7 +217,12 @@ if (portletTitleBasedNavigation) {
 
 							<%
 							try {
-								wikiEngineRenderer.renderEditPageHTML(selectedFormat, pageContext, node, wikiPage);
+								if (wikiPage.isNew() && (templatePage != null)) {
```

Comment:
Hey @jonathanmccann @Alec-Shay I think that wikiPage could be null right? What should we do in that case? Can you please resend considering that scenario?
Thanks!